src/Runner.js

```diff
@@ -78,6 +78,9 @@ export default class Runner extends EventEmitter {
     if (this.options.noColor === true) {
       args.push('--no-color');
     }
+    if (this.options.reporters) {
+      args.push('--reporters', this.options.reporters);
```

Comment:
see [jest cli](https://jestjs.io/docs/en/cli#--reporters), `this.options.reporters` probably should be an array and we will need to pass each with its own `--reporters` flag.

Reply:
Good idea. Done.